pkg/odo/cli/component/create.go

```diff
@@ -466,7 +466,7 @@ func (co *CreateOptions) Validate() (err error) {
 	}
 
 	if !supported {
-		log.Infof("Warning: %s is not fully supported by odo, and it is not guaranteed to work.", *co.componentSettings.Type)
+		log.Infof("Warning: %s is not fully supported by odo, and it is not guaranteed to work", *co.componentSettings.Type)
```

Comment:
https://travis-ci.com/openshift/odo/jobs/251973014#L9431
Please add the `.` at the end or remove it from the test